Marlin/src/module/endstops.cpp

```diff
@@ -259,8 +262,15 @@ void Endstops::not_homing() {
 #if ENABLED(VALIDATE_HOMING_ENDSTOPS)
   // If the last move failed to trigger an endstop, call kill
   void Endstops::validate_homing_move() {
-    if (trigger_state()) hit_on_purpose();
-    else kill(GET_TEXT_F(MSG_KILL_HOMING_FAILED));
+    if (trigger_state())
+      hit_on_purpose();
+    else {
+      #if ENABLED(SOVOL_SV06_RTS)
+        rts.gotoPage(55, 110);
```

Comment:
We should create an `enum` for all these Page IDs because "mystery meat" numbers are not easy to maintain.